modules/apps/foundation/frontend-js/frontend-js-spa-web/src/main/java/com/liferay/frontend/js/spa/web/configuration/SPAConfiguration.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.js.spa.web.configuration;
+
+import aQute.bnd.annotation.metatype.Meta;
+
+import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
+
+/**
+ * @author Bruno Basto
+ */
+@ExtendedObjectClassDefinition(category = "foundation")
+@Meta.OCD(
+	id = "com.liferay.frontend.js.spa.web.configuration.SPAConfiguration",
+	localization = "content/Language", name = "spa.configuration.name"
+)
+public interface SPAConfiguration {
+
+	@Meta.AD(
+		deflt = "-1", description = "spa.cache.expiration.description",
+		name = "spa.cache.expiration.name", required = true
+	)
+	public String cacheExpirationTime();
+
+	@Meta.AD(
+		deflt = "/documents|/image",
+		description = "spa.excluded.paths.description",
+		name = "spa.excluded.paths.name", required = false
+	)
+	public String[] excludedPaths();
```

Comment:
I removed that because I was unsure we want the system admin to change the default values for /documents and /image. They are mapped in web.xml (if I remember correctly) and thus require code changes to work. What we can do  is make those values fixed and add the ability for the sys admin to specify more excluded paths. What do you think?